docs/user/app_getting_started.md

```diff
@@ -34,13 +34,13 @@ Clicking this button will open a standard Nautobot create/edit form. This form a
 
 - **Tile Width** and **Tile Depth**:
   Define the relative proportions of each tile when rendered in the Nautobot UI.
-    - You can use the default settings for a square grid.
-    - Alternatively, customize these parameters for a rectangular grid.
+  - You can use the default settings for a square grid.
+  - Alternatively, customize these parameters for a rectangular grid.
 
 - **Movable Tiles**:
   Determine if you want tiles to be movable once placed.
-    - This feature is a optional setting to assist with Custom Label creation
-    - Default: `"True"`
+  - This feature is a optional setting to assist with Custom Label creation
+  - Default: `"True"`
```

Comment:
This should be reverted, the depth does not display as intended:

Existing:
![image](https://github.com/user-attachments/assets/93952252-9585-4c70-a28a-cb2de847327b)

With these changes:
![image](https://github.com/user-attachments/assets/63975c1d-0a32-461c-9c23-c9fb8e562839)